tests/utils.spec.js

```diff
@@ -24,3 +24,10 @@ test('getComponentName > can handle multiple scenarios to determine component na
     element.removeAttribute('role')
     expect(getComponentName(element)).toBe('div')
 })
+
+test('createComponentId will return a 13 digit number', async () => {
+    const id = createComponentId()
+
+    expect(typeof id).toBe('number')
+    expect(id.toString()).toHaveLength(13)
```

Comment:
🤔 surely there are libraries/tools that will create a more compact and collision-resistant id/digest

Maybe something like a short sha of the component markup using the Web Crypto API (https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto/digest#Examples)?

Reply:
> The reason I'm saying this is that a 13 character id will be generated, encoded & passed from backend -> devtools for each component, which can add up in terms of encoding time/message size.

What would be the difference if we used another way to generate them?

I don't mind though, will give me an excuse to understand the `SubtleCrypto` API :D

Reply:
Oh I see, sliced to 5 characters. Yeah, I'll check. 

Do you think it would be uncommon to have 1000+ components on a page?

Reply:
> Oh I see, sliced to 5 characters. Yeah, I'll check.
> 
> Do you think it would be uncommon to have 1000+ components on a page?

yeah so the first 5 characters of the sha256 of something, we might need to decide what the "something" is though. I think we can also use the Crypto API to generate a random sequence.

1000+ components is definitely in "uncommon" land

Reply:
Do you see any reason not to just use a counter that starts at 1?

Reply:
Can't think of any

Reply:
How about like this? A simple counter seems to be the most reliable and smallest impact on the payload.

Reply:
🤷🏼‍♀️ seems fine to me, my worry with a counter is that there's some big reason it doesn't but we don't think of it 😂😂

Reply:
I think it's fine. Vue [uses a counter](https://github.com/vuejs/vue/blob/dev/src/core/instance/init.js#L20) to track components. Not in devtools, but since Alpine doesn't add any identifiers (maybe it should) I think we're okay adding it.

Reply:
Yeah also, since backend.js runs in the specific window, I think that's fine